src/extension/monitor/core/src/main/java/org/geoserver/monitor/ows/MonitorCallback.java

```diff
@@ -96,7 +96,7 @@ public Operation operationDispatched(Request request, Operation operation) {
             }
         }
 
-        monitor.update();
+        // monitor.update();
```

Comment:
For the calls to monitor.update() in the MonitorCallback, I see them calls in MonitorFilter which call with basically the same information.  

I'm wondering if it makes sense to think of MonitorCallback as just setting data and rely on MonitorFilter for calls which would send data to the Monitor implementations?